tests/hierarchy_driver.cc

```diff
@@ -20,6 +20,7 @@
 template <int dim, int fe_degree>
 void matrix_free_two_grids(std::shared_ptr<boost::property_tree::ptree> params)
 {
+  const bool test_preconditioner = params->get("is preconditioner", true);
```

Comment:
left const. I thought you were one of us...

Reply:
Just try
```
 grep -rIn "  const " source/ include/ tests/
```